website_user_login_redirect_custom/controllers/main.py

```diff
@@ -0,0 +1,67 @@
+# © 2025 Cetmix OU
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import str2bool
+
+# Import the base web controller (always available)
+from odoo.addons.web.controllers.main import Home
+
+
+class WebsiteRedirectCustom(Home):
+    """
+    Inherit from Home to override login behavior
+    This works even without auth_signup module
+    """
+
+    def _should_redirect_custom(self) -> bool:
+        """Check if we should perform custom redirect"""
+        if not request.session.uid:
+            return False
+
+        # Do not redirect backend/internal users
+        if request.env.user.has_group("base.group_user"):
+            return False
+
+        enabled = (
+            request.env["ir.config_parameter"]
+            .sudo()
+            .get_param("website_user_login_redirect_custom.enabled", "False")
+        )
+        return bool(str2bool(enabled))
+
+    def _get_custom_redirect_url(self):
+        """Get custom redirect URL"""
+        return (
+            request.env["ir.config_parameter"]
+            .sudo()
+            .get_param("website_user_login_redirect_custom.url", "/")
+        )
+
+    @http.route("/web/login", type="http", auth="public", website=True, sitemap=False)
+    def web_login(self, redirect=None, **kw):
+        """Override standard login route to apply redirect after login."""
+        response = super(WebsiteRedirectCustom, self).web_login(redirect=redirect, **kw)
+
+        # Only redirect if:
+        # 1. No explicit redirect parameter was provided
+        # 2. User is logged in
+        # 3. Feature is enabled
+        # 4. We have a valid redirect response
+        # 5. Custom URL is valid
+        if (
+            not redirect
+            and self._should_redirect_custom()
+            and hasattr(response, "status_code")
+            and response.status_code in (303, 302)
+        ):
+            custom_url = self._get_custom_redirect_url().strip()
```

Comment:
_get_custom_redirect_url must return the URL already prepared to be used in redirect OR False